esmvaltool/cmorizers/obs/utilities.py

```diff
@@ -137,6 +137,7 @@ def flip_dim_coord(cube, coord_name):
     if coord.bounds is not None:
         coord.bounds = np.flip(coord.bounds, axis=0)
     cube.data = da.flip(cube.core_data(), axis=coord_idx)
+    return cube
```

Comment:
no need to return the cube, a function that operates on the cube will automatically return the cube without the explicit return call

Reply:
I did not see the changes made reflected if I did not return the cube. Interestingly, the cube coord bounds for latitude did not get set in a standard fashion. I am looking into why now.

Reply:
@rswamina Can you change this back, as @valeriupredoi suggested?